[2023-03-16 16:11:02,097] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task manual__2023-03-16T07:10:58.673770+00:00 [queued]>
[2023-03-16 16:11:02,100] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task manual__2023-03-16T07:10:58.673770+00:00 [queued]>
[2023-03-16 16:11:02,100] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-16 16:11:02,100] {taskinstance.py:1239} INFO - Starting attempt 1 of 4
[2023-03-16 16:11:02,100] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-16 16:11:02,102] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): second_task> on 2023-03-16 07:10:58.673770+00:00
[2023-03-16 16:11:02,103] {standard_task_runner.py:52} INFO - Started process 99122 to run task
[2023-03-16 16:11:02,106] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'manual__2023-03-16T07:10:58.673770+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T/tmp6e1wu_9r', '--error-file', '/var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T/tmpq_ei7e0g']
[2023-03-16 16:11:02,106] {standard_task_runner.py:77} INFO - Job 7: Subtask second_task
[2023-03-16 16:11:02,123] {logging_mixin.py:109} INFO - Running <TaskInstance: our_first_dag_v2.second_task manual__2023-03-16T07:10:58.673770+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2023-03-16 16:11:02,137] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=byeonghwa
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-16T07:10:58.673770+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-16T07:10:58.673770+00:00
[2023-03-16 16:11:02,137] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/x1/7qlvjnbj06q1nkyhy70vbb2c0000gn/T
[2023-03-16 16:11:02,138] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, i am task2 and will be running after task1']
[2023-03-16 16:11:02,141] {subprocess.py:85} INFO - Output:
[2023-03-16 16:11:02,143] {subprocess.py:89} INFO - hey, i am task2 and will be running after task1
[2023-03-16 16:11:02,143] {subprocess.py:93} INFO - Command exited with return code 0
[2023-03-16 16:11:02,148] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20230316T071058, start_date=20230316T071102, end_date=20230316T071102
[2023-03-16 16:11:02,168] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-16 16:11:02,174] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
